### PR TITLE
Update: Install new design-system version and restrict routes

### PR DESCRIPTION
**Pull request by cmarin001**
This PR focuses on updating the linpar application to the latest version of our library and temporarily restricting access to all routes except for the home route. This is a strategic move to ensure a smooth transition and testing phase as we integrate the latest features and improvements from our library.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -17,12 +17,12 @@ const router = createBrowserRouter(
   createRoutesFromElements(
     <>
       <Route path="/" element={<Home />} errorElement={<ErrorPage />} />
-      <Route path="privileges/*" element={<PrivilegesRoutes />} />
+      {/* <Route path="privileges/*" element={<PrivilegesRoutes />} />
```
Comment (Andresbl123) [inline src/App.tsx:20]: Al comentar esto hay cosas que ya no se utilizan en las importaciones línea 12 a 14

**Comment by Andresbl123** [comment]
Validar el porque genera error en los test de githab

File: src/App.tsx (hunk 1 above)
Comment (cmarin001) [reply src/App.tsx:20]: image

**Review by Andresbl123 — APPROVED** [review]
cumple requerimientos

**Pull request merged**